### ISSUE TITLE
Add new status to identify a PR is attached to the bug

### ISSUE THREAD

**Issue by punamdahiya** [k88hudson/bugzy#79]
Scope of this issue is to check in Bugzilla if a PR is attached to the bug and update status/lifecycle to 'InReview'

**Comment by punamdahiya** [comment]
Depends on #102

**Comment by punamdahiya** [comment]
One possible to figure if a github PR is attached to bug is by checking if GitHub Bugzilla PR link account has a comment in the bug

**Comment by k88hudson** [comment]
Emma suggested this as a stopgap:

https://bugzilla.mozilla.org/buglist.cgi?chfield=%5BBug%20creation%5D&chfieldfrom=2018-12-10&chfieldto=Now&classification=Client%20Software&classification=Developer%20Infrastructure&classification=Components&classification=Server%20Software&classification=Other&f1=attachments.ispatch&f2=component&o1=equals&o2=substring&v1=true&v2=Activity%20Stream&list_id=14634926

**Comment by AllegroFox** [comment]
closing as a duplicate of https://github.com/k88hudson/bugzy/issues/187
